app/views/recipes/_food_recipe_item.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="foodTableBody">
+<div><%= food_item.food.name%></div>
+<div><%= food_item.quantity%> <%= food_item.food.measurement_unit%></div>
+<div>$<%= food_item.food.price*food_item.quantity%></div>
+<div>Delete <%= food_item.id%></div>
+</div>
```

Comment:
While you did a good job working on the project I noticed the delete button in your Foods tab and general shopping list is not functional. It would be nice to implement functionality for the delete action so a user is able to delete a given food list

![Screenshot 2023-12-02 at 11 41 02 PM](https://github.com/Graciano1997/recipe_app/assets/79506105/2094e65d-989c-4d9d-885b-d9c2323cdff8)
![Screenshot 2023-12-02 at 11 48 34 PM](https://github.com/Graciano1997/recipe_app/assets/79506105/d2c7c762-50bd-4460-919f-72237e1e93aa)